scripts/fuzz_opt.py

```diff
@@ -178,6 +196,12 @@ def randomize_fuzz_settings():
     os.path.join('lit', 'passes', 'optimize-instructions-gc.wast'),
     os.path.join('lit', 'passes', 'inlining_splitting.wast'),
 ]
+
+if shared.options.auto_initial_contents:
+    IMPORTANT_INITIAL_CONTENTS += auto_select_recent_initial_contents()
```

Comment:
It might be nice to add some logging of what files were selected here. But it would be hidden fast by other output, so maybe it should pause for user input to continue or something like that..?